cppcache/integration/test/Position.cpp

```diff
@@ -41,19 +41,19 @@ Position::Position(std::string id, int32_t outstandingShares) {
 
 void Position::init() {
```

Comment:
Good C++ practice would sanitize this to an list of initializations on a common constructor. `std::string` initializes to `""` already.

Reply:
Good catch. Done.